cmd/kubeadm/app/apis/kubeadm/env.go

```diff
@@ -31,7 +33,7 @@ func SetEnvParams() *EnvParams {
 
 	envParams := map[string]string{
 		"kubernetes_dir":  "/etc/kubernetes",
-		"host_etcd_path":  "/var/lib/etcd",
+		"host_etcd_path":  v1alpha1.DefaultEtcdDataDir,
```

Comment:
No problem, I appreciate the feedback 👍  does this commit address what you mean? https://github.com/kubernetes/kubernetes/pull/42246/commits/4b065f8a27a08d70b0e39b2ef99af44f35689859